### PR TITLE
Added a link

### PR DESCRIPTION
**Pull request by joinimran**
I have added a link to autopilot reset article where the user can use REagentC command to fix the issues.

### CODE REVIEW

**Comment by joinimran** [comment]
Problem for this PR is https://github.com/MicrosoftDocs/windows-itpro-docs/issues/3071

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/windows-autopilot/troubleshooting.md [hunk 1]
```diff
@@ -92,3 +92,5 @@ Error code 801C0003 will typically be reported on an error page titled "Somethin
 See [this knowledge base article](https://support.microsoft.com/help/4089533/troubleshooting-windows-device-enrollment-problems-in-microsoft-intune) for assistance with Intune enrollment issues.  Common issues include incorrect or missing licenses assigned to the user or too many devices enrolled for the user.
 
 Error code 80180018 will typiclaly be reported on an error page titled "Something went wrong."  This error means that the MDM enrollment failed.
```
Comment (nenonix) [inline windows/deployment/windows-autopilot/troubleshooting.md]: Error code 80180018 will **typically** be reported on an error page titled "Something went wrong**".**

File: windows/deployment/windows-autopilot/troubleshooting.md [hunk 2]
```diff
@@ -92,3 +92,5 @@ Error code 801C0003 will typically be reported on an error page titled "Somethin
...
+
+If Autopilot reset failed immeidately with an error "Ran into trouble. Please sign-in with an administrator account to see why and reset manually" See [this article](https://docs.microsoft.com/en-us/education/windows/autopilot-reset#troubleshoot-autopilot-reset) for more help.
```
Comment (nenonix) [inline windows/deployment/windows-autopilot/troubleshooting.md]: **If Autopilot Reset fails immediately with an error: "Ran into trouble. Please sign-in with an administrator account to see why and reset manually" see [Troubleshoot Autopilot Reset](https://docs.microsoft.com/education/windows/autopilot-reset#troubleshoot-autopilot-reset) for more help.**

Comment (nenonix) [inline windows/deployment/windows-autopilot/troubleshooting.md]: If **Autopilot Reset fails immediately**** with an error...

**Comment by illfated** [comment]
Ouch. Looks like the merge happened just a tad too quick for the last review to be included.
I guess it will need a new PR or direct write-up by a team member to get fixed.

**Comment by nenonix** [comment]
Yes, @illfated the review process is new and we are still figuring it out.  Apologies for the snarl-up @lizap -we will speed up and settle down soon (I hope). I will submit a new PR to fix the remaining issues.

**Pull request merged**